CHANGELOG.md

```diff
@@ -0,0 +1,30 @@
+# Labrador Version History
+
+Labrador v0.3 (16 March 2021)
+-------------------------------------------------
+Labrador v0.3 is a minor update incorporating the change described below:
```

Comment:
Yeah, I'm not sure that you get away quite as easily as that 😅 

Here is a list of the changes since version 0.2 - https://github.com/ewels/Labrador/compare/v0.2...master - the commit messages seem to be mostly pretty descriptive, so you could maybe just get that and prune out the cruft.